exercises/practice/resistor-color-duo/resistor-color-duo.ua

```diff
@@ -0,0 +1 @@
+ResistorColorDuo ← |1 ⊙(⍤ "Please implement ResistorColorDuo")
```

Comment:
The prob-specs definition uses` value` as the name of the property. Maybe change it to that (or something like it)?

Reply:
Also, it would be good to include the comments as in https://github.com/exercism/uiua/pull/29